packages/composer/test/proxy.test.js

```diff
@@ -458,7 +458,8 @@ test('should proxy all services if none are defined', async t => {
   }
 })
 
-test('should fix the path using the referer only if asked to', async t => {
+// TODO(mcollina): this now segfaults. We should investigate why and fix it in core.
+test('should fix the path using the referer only if asked to', { skip: true }, async t => {
```

Comment:
@ShogunPanda ptal

Reply:
ACK. Until we get some time to investigate there's really nothing we can do.